app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -192,15 +192,19 @@ def create_valkyrie_work
     def update_valkyrie_work
       form = build_form
       return after_update_error(form_err_msg(form)) unless form.validate(params[hash_key_for_curation_concern])
-
       result =
         transactions['change_set.update_work']
-        .with_step_args('work_resource.add_file_sets' => { uploaded_files: uploaded_files, file_set_params: params[hash_key_for_curation_concern][:file_set] })
+        .with_step_args('work_resource.add_file_sets' => { uploaded_files: uploaded_files, file_set_params: params[hash_key_for_curation_concern][:file_set] },
+                        'work_resource.update_work_members' => { work_members_attributes: work_members_attributes })
```

Comment:
Can a new work have existing members added?  I wonder if there should be something similar for valkyrie create?

Reply:
Lynette says this can be moved to a new ticket.